### PR TITLE
vv-01 Add fix

### PR DESCRIPTION
**Pull request by AntipMen**
In this PR was added some changes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,7 +3,8 @@
 	<head>
 		<meta charset="UTF-8" />
 		<link rel="stylesheet" href="./styles.css" />
-		<title>PR test</title>
+
```
Comment (V-tyna) [inline index.html]: Please delete space.

File: index.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
 const root = document.getElementById('root');
 
 root.style.backgroundColor = 'blue';
+console.log('sho', root)
```
Comment (V-tyna) [inline index.js]: Console.log is redundant.

**Review by V-tyna — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (AntipMen) [reply index.html]: Sure. Thanks for noticing.

File: index.js (hunk 2 above)
Comment (AntipMen) [reply index.js]: Fixed. Thank you!

**Review by V-tyna — APPROVED** [review]

**Pull request merged**